internal/proto/local/controller/api/resources/hostsets/v1/host_set.proto

```diff
@@ -48,9 +48,21 @@ message HostSet {
 	repeated string host_ids = 100 [json_name="host_ids"];
 
 	// An ordered list of endpoint preferences used to choose from among
-	// multiple possible endpoints for a host.
+	// multiple possible endpoints for a host. Preferences are specified by
+	// "cidr:<valid IPv4/6 CIDR>" or "dns:<globbed name>", specifying which IP
+	// address or DNS name out of a host's available possibilities should be
+	// preferred. If no preferences are specified, a value will be chosen _at
+	// random_ from all available values. May not be valid for all plugin types.
```

Comment:
A value won't be selected at random though, right?  Don't we still prioritize private ipv4 addresses, etc?

Reply:
Copied and pasted from elsewhere, need to update it clearly.

Reply:
Fixed.